### DISCUSSION TITLE
文件外链现在打开是直接下载，可以增加一个在线预览的吗

### DISCUSSION THREAD

**Discussion by zzkuner** [cloudreve/cloudreve#3206] [General]
【功能请求】
有时候只是想使用把这个文件分享给别人看，但是别人还得下载，可以像kkview那样在线预览文件吗，感觉这个还是很重要的，就是打开就是那个文件。
 https://kkview.cn/zh-cn/index.html

**Comment by SANYE-CH** [comment]
外链就是这样的 建议你使用分享
